### PR TITLE
MRG: Deprecate frequencies -> freqs

### LINKED ISSUE #4116: Consistency in naming freqs vs. frequencies
It looks like most of the TFR-style functionality calls it `freqs`:

https://github.com/mne-tools/mne-python/search?utf8=%E2%9C%93&q=freqs&type=

while the stats and connectivity style objects call it `frequencies`:

https://github.com/mne-tools/mne-python/search?utf8=%E2%9C%93&q=frequencies&type=

Should we standardize this?

### PR DESCRIPTION
**Pull request by larsoner**
I actually hit this case today, so I figured I'd start a PR to fix #4116.

~~WIP because I've only fixed one, and haven't updated any tests.~~

Closes #4116.

### CODE REVIEW

**Comment by larsoner** [comment]
Okay this one should be good to go now

File: mne/decoding/time_frequency.py [hunk 1]
```diff
@@ -81,6 +82,11 @@ def __init__(self, frequencies, sfreq=1.0, method='morlet', n_cycles=7.0,
         self.n_jobs = n_jobs
         self.verbose = verbose
 
+    @property
+    def frequencies(self):
+        """Deprecated and will be removed in 0.16. use freqs."""
```
Comment (agramfort) [inline mne/decoding/time_frequency.py]: you don't warn

File: mne/time_frequency/csd.py [hunk 2]
```diff
@@ -28,25 +28,31 @@ class CrossSpectralDensity(object):
...
         self.ch_names = cp.deepcopy(ch_names)
         self.projs = cp.deepcopy(projs)
         self.bads = cp.deepcopy(bads)
-        self.frequencies = np.atleast_1d(np.copy(frequencies))
+        self.freqs = np.atleast_1d(np.copy(freqs))
         self.n_fft = n_fft
 
+    @property
+    def frequencies(self):
+        """Deprecated and will be removed in 0.16. Use freqs."""
```
Comment (agramfort) [inline mne/time_frequency/csd.py]: same here

File: mne/time_frequency/csd.py [hunk 3]
```diff
@@ -150,8 +156,8 @@ def csd_epochs(epochs, mode='multitaper', fmin=0, fmax=np.inf,
     sfreq = epochs.info['sfreq']
     orig_frequencies = fftfreq(n_fft, 1. / sfreq)
```
Comment (agramfort) [inline mne/time_frequency/csd.py]: orig_frequencies -> orig_freqs

**Comment by larsoner** [comment]
Rebased, comments, addressed, and fixed a couple unnecessary general FFT -> RFFT uses

**Comment by agramfort** [comment]
appveyor is not happy

**Comment by larsoner** [comment]
Latest commit should make CIs happy, and it fixes a bug in our multitaper PSD computations having to do with DC and Nyquist terms.

**Comment by agramfort** [comment]
thx @larsoner

**Pull request merged**